### PR TITLE
update select dropdown style [SATURN-1208]

### PR DESCRIPTION
**Pull request by zarsky-broad**
Worked with Joy; verified in UI.

### CODE REVIEW

File: src/components/common.js [hunk 1]
```diff
@@ -276,15 +276,26 @@ const commonSelectProps = {
     singleValue: base => ({ ...base, color: colors.dark() }),
     option: (base, { isSelected, isFocused, isDisabled }) => _.merge(base, {
       overflowWrap: 'break-word',
-      backgroundColor: isSelected ? colors.light(0.4) : isFocused ? colors.dark(0.15) : undefined,
-      color: isSelected ? colors.accent() : isDisabled ? undefined : colors.dark(),
+      fontWeight: isSelected ? 600 : undefined,
+      backgroundColor: isFocused ? colors.dark(0.15) : 'white',
```
Comment (ehanna4) [inline src/components/common.js:280]: nit: I haven't seen 'white' as a color used much in the app, should this be 'colors.light' instead?

Comment (petesantos) [reply src/components/common.js:280]: +1 The hard coded color may make it more difficult to change the theme in the future. Especially if adding a high contrast theme for a11y - the hardcoded color may make this more difficult

**Review by petesantos — APPROVED** [review]
Looks good - much more intuitive.
Brought up PR locally tested changes.

File: src/components/common.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/common.js]: I see 66 uses of `white` vs just 27 uses of `light` in the code, I'm going to leave this as-is for now and we can consider a wider refactor later.

Comment (zarsky-broad) [reply src/components/common.js:280]: I see 66 uses of `white` vs just 27 uses of `light` in the code, I'm going to leave this as-is for now and we can consider a wider refactor later.

**Pull request merged**